packages/user-service/jest.config.js

```diff
@@ -6,6 +6,8 @@ module.exports = {
   ],
   "transform": {
     "^.+\\.(ts|tsx|js)$": "ts-jest"
+    ,
```

Comment:
Remove the extra newline here. Move this comma to the above line.

Reply:
Removed the new line